### PR TITLE
2022.4.1 Release

### PR DESCRIPTION
**Pull request by ayushdg**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ayushdg** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/dask-contrib/dask-sql/archive/{{ version }}.tar.gz
-  sha256: 1f9438ed71d60d534e3fd9df13d09c99c7056f600d697d119633c732f92f70a9
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask_sql-{{ version }}.tar.gz
+  sha256: ce1d19dc05c30b7bb505da3c7798dcc70f3f1b11fc2b56b7bd5ec4196b41b863
```
Comment (jakirkham) [inline recipe/meta.yaml:10]: Did we fix the Java source code packaging issue @charlesbluca?

Comment (charlesbluca) [reply recipe/meta.yaml:10]: Yup, verified that the latest PyPI tars contain the `planner` folder required for builds

File: recipe/meta.yaml [hunk 2]
```diff
@@ -24,24 +24,25 @@ requirements:
...
-    - tzlocal >=2.1
+    - nest-asyncio
+    - pandas >=1.0.0
     - prompt-toolkit
     - pygments
-    - nest-asyncio
+    - python >=3.8
     - tabulate
+    - tzlocal >=2.1
+    - uvicorn >=0.11.3
```
Comment (charlesbluca) [inline recipe/meta.yaml:38]: @jakirkham for context, we are considering reordering the packages in the way generated by grayskull to hopefully make package diffs easier to parse moving forward - LMK if that makes sense to you 🙂

**Review by charlesbluca — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/meta.yaml:38]: Yeah that's what I gathered. Seems ok

Also regarding dependency ordering, there are a few relevant upstream issues: 
* https://github.com/conda-incubator/grayskull/issues/259
* https://github.com/conda-incubator/grayskull/issues/288

**Review by galipremsagar — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (charlesbluca) [reply recipe/meta.yaml:38]: Cool, thanks for sharing - was interested in how those issues were being recieved

**Pull request merged**